commit e56dc9e2949edff7932474f2552dd134734cc857
Author: J. Bruce Fields <bfields@fieldses.org>
Date:   Thu Jul 30 20:33:57 2020 -0400

    nfsd: remove fault injection code
    
    It was an interesting idea but nobody seems to be using it, it's buggy
    at this point, and nfs4state.c is already complicated enough without it.
    The new nfsd/clients/ code provides some of the same functionality, and
    could probably do more if desired.
    
    This feature has been deprecated since 9d60d93198c6 ("Deprecate nfsd
    fault injection").
    
    Signed-off-by: J. Bruce Fields <bfields@redhat.com>

diff --git a/Documentation/admin-guide/nfs/index.rst b/Documentation/admin-guide/nfs/index.rst
index 6b5a3c90fac5..3601a708f333 100644
--- a/Documentation/admin-guide/nfs/index.rst
+++ b/Documentation/admin-guide/nfs/index.rst
@@ -12,4 +12,3 @@ NFS
     nfs-idmapper
     pnfs-block-server
     pnfs-scsi-server
-    fault_injection